airflow/models/__init__.py

```diff
@@ -3118,6 +3150,50 @@ def date_range(self, start_date, num=None, end_date=timezone.utcnow()):
             start_date=start_date, end_date=end_date,
             num=num, delta=self._schedule_interval)
 
+    def create_edges(self, graph_id):
```

Comment:
Can we prefix with underscore if this isn't meant to be exposed to consumers of this class? i.e., `_create_edges`

Reply:
fixed

Reply:
now This is public again because it's also used outside of this class